readme/mobile.md

```diff
@@ -1,3 +1,4 @@
 # Mobile app
+<img src="https://github.com/abhi-bhatra/joplin/blob/patch-3/Assets/JoplinMobileui.jpg" style="max-width=100%"/>
```

Comment:
You are still pointing to your repository.

Reply:
And the file should be renamed to `JoplinMobileUI.jpg` and placed in `Assets/WebsiteAssets/images/`.